packages/tinacms/src/toolkit/fields/components/reference/reference-select.tsx

```diff
@@ -95,55 +115,93 @@ const useGetOptionSets = (cms: TinaCMS, collections: string[]) => {
   return { optionSets, loading }
 }
 
-const ReferenceSelect: React.FC<ReferenceSelectProps> = ({
+//function to get filename from optionSets for display text in combobox
+//file name is used for display text because title can be nullable (user can defined name rather than title field)
```

Comment:
```suggestion
// function to get the filename from optionSets to display text in the combobox
// file name is used to display text as the title can be nullable (user can define the name rather than title field)
//? Note - This is looking for a field with `name` or `title`
```